### ISSUE TITLE
R HE hackathon (meta-issue)

### ISSUE THREAD

**Issue by seabbs** [giabaio/BCEA#6]
Hi @giabaio, 

TLDR: Sorry for the huge number of issues. All are suggestions so feel free to ignore/close. Equally if you are interested but have no time then I am happy to do some work on these.

I've been attending an R health economics hackathon and been reviewing your package to see if more tooling is needed for cost effectiveness analysis (it's not - your package is great!). We actually started on a [package](https://github.com/seabbs/ceplotr) and realised very shortly after finishing the infrastructure that yours did everything we were considering doing.

That being said we identified a few things that we thought could be changed to improve the user experience for people using BCEA with non-traditional models. I am going to open a few issues highlighting some of our thoughts but feel free to close any/all of them if you think it isn't required. 

Very happy to work on any or all of these issues if that would be helpful - just let me know :smile:

**Comment by giabaio** [comment]
Thanks for all the messages, @seabbs! First of all, thanks for your kind words on our work. Also, I'm very happy to have people collaborate & contribute to `BCEA`, so absolutely --- I'll reply to your messages individually and close the ones that don't require further interactions (eg this one). Of course, feel free to re-open if you think otherwise, though!
